.github/workflows/e2e-ci-playwright.yml

```diff
@@ -0,0 +1,278 @@
+name: E2e CI Playwright Tests
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+    inputs:
+      browser:
+        description: 'Browser to run tests on'
```

Comment:
Question maybe for the future. Should we leverage the ability of playwright to run tests on firefox and webkit too and run all the tests there in parallel? Moving them inside the test matrix?

Reply:
Yes, we can even run the tests in parallel against all the pre-bundled browsers with a small configuration change in the execution command, without explicitly specifying any particular browser. Probably we will revisit inputs definition i.e. remove the input parameters and run always with all supported browsers while we include the strategy to run all browsers.